### ISSUE TITLE
Create dedicated ssh key automatically for each host

### ISSUE THREAD

**Issue by moul** [moul/advanced-ssh-config#37]
Dynamically create a new ssh key pair for each host and submit it on the first connection

**Comment by Skrupellos** [comment]
I had such a setup quit a time ... would not do again.

1. There is [no security benefit](https://security.stackexchange.com/a/40061).
2. SSH Server only allow n (3 or 4) authentication attempts with public keys. Therefore you have to define for each host which key to use, otherwise >3 keys are tried and you can't login. You also have to keep track of which key is on which machine (to revoke the correct key).
3. As a server admin, it's not nice having users, who have a bunch of keys, which makes it difficult to identify/revoke a single user.

In conclusion, I would suggest to close this Issue and stick to #94

**Comment by moul** [comment]
Looks good to me
